### ISSUE TITLE
[Issue] Streams failing at ad breaks

### ISSUE THREAD

**Issue by adamef93** [tonywagner/EPlusTV#71]
Me again haha. This just started happening tonight as far as I can tell. I recreated the container, logged back in, and tested playing back the m3u link in VLC and through XTeve to see if it was actually an issue with EPlus or not. The error message that's logged is quite long and was hard to copy in its entirety but I think I got the part that matters toward the end:

```
  cause: Error: connect ECONNREFUSED 0.0.0.0:443
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '0.0.0.0',
    port: 443
  }
}
```
Also, even without the above error block, all the channels disconnect whenever there's a break:

`Could not get a playlist for channel #9063. Please make sure there is an event scheduled and you have access to it`

**Comment by m0ngr31** [comment]
What are your running variables?

**Comment by adamef93** [comment]
- TZ=America/New_York
      - ESPNPLUS=true
      - START_CHANNEL=9050

Did you mean [this reply](https://github.com/m0ngr31/EPlusTV/issues/61#issuecomment-1745423933) here? Haha

**Comment by m0ngr31** [comment]
I'm curious if it's breaking xteve. If you play it straight to VLC does it break?

**Comment by adamef93** [comment]
I feel like I tried that and it didn't work but I'll try again later. I also don't typically run EPlus through Xteve and only did it for testing

Unrelated (and contradictory haha) - I actually am running EPlus through Xteve now because I'm running EPlus through a VPN container to bypass local blackouts. For some reason, this seems to break streams unless proxied through Xteve. But I'm sure that's a whole different can of worms. Just wanted to mention it

---

Edit: Honestly, disregard the part about the VPN and Xteve. I just tested streams on both an EPlus native m3u and an Xteve m3u and certain streams worked on both and broke on both. Idk
